Tools/RemoteConsole/ly_remote_console/ly_remote_console/remote_console_commands.py

```diff
@@ -82,12 +84,12 @@ def _default_on_message_received(raw):
 
 
 def _default_disconnect():
-    # type: () -> None
     """
     On a disconnect a user can overwrite the functionality with any function, this one will just print to the
     logger a line 'Disconnecting from the Port.'
     :return: None
     """
+    print('Disconnecting')
     logger.info('Disconnecting from the Port')
```

Comment:
Duplicate line, let's not do this.

Reply:
Visual feedback, loggers logs to a file silently.